spec/models/pacbio/library_spec.rb

```diff
@@ -18,12 +18,29 @@
     expect(build(:pacbio_library, fragment_size: nil)).to_not be_valid
   end
 
-  it 'must have a tag' do
-    expect(build(:pacbio_library, tag: nil)).to_not be_valid
-  end
-
   it 'will have a uuid' do
     expect(create(:pacbio_library).uuid).to be_present
   end
-  
+
+  context 'wells' do
+    it 'can have one or more' do
+      library = create(:pacbio_library)
+      library.wells << create_list(:pacbio_well, 5)
+      expect(library.wells.count).to eq(5)
+    end
+  end
+
+  context 'requests' do
+
+    let(:library) { create(:library)}
+
+    it 'can have one or more' do
+      library = create(:pacbio_library)
+      (1..5).each do |i|
+        create(:pacbio_request_library, request: create(:pacbio_request), library: library, tag: create(:tag))
+      end
+      expect(library.requests.count).to eq(5)
+    end
+  end
+
```

Comment:
Could also add tests for `well_libraries` and `request_libraries`

Reply:
I did think about that but the well_libraries is purely a join table and I thought the fact that you can add libraries to a well proves that well_libraries are working correctly. What do you think?
With request_libraries again I did think about that. I have added tests below and the fact that a library has requests proves it is working correctly. Happy to add them if you think it is necessary.

Reply:
Think you're right and `libraries` proves the join is working as expected, same with request_libraries